h2o-docs/src/product/cloud-integration/ec2-and-s3.rst

```diff
@@ -59,7 +61,53 @@ When running H2O in standalone mode using the simple Java launch command, we can
         set_s3_credentials("AWS_ACCESS_KEY", "AWS_SECRET_KEY")
         h2o.import_file(path = "s3://bucket/path/to/file.csv")
 
-**Note**: Passing credentials in the URL, e.g. ``h2o.importFile(path = "s3://<AWS_ACCESS_KEY>:<AWS_SECRET_KEY>@bucket/path/to/file.csv")``, is considered a security risk and is deprecated. 
+        
+-  Just like regular AWS credentials, temporary credentials using AWS SESSION TOKEN can be passed in standalone mode by creating a ``core-site.xml`` file and passing it in with the flag ``-hdfs_config``. For an example ``core-site.xml`` file, refer to `Core-site.xml`_. The only difference lies in specifying a triplet of (AWS SECRET KEY, AWS SECRET ACCESS KEY and AWS SESSION TOKEN)
+
+   1. Edit the properties in the core-site.xml file to include your Access Key ID, Access Key, and Session Token as shown in the following example:
+
+     ::
+
+       <property>
+         <name>fs.s3.awsAccessKeyId</name>
+         <value>[AWS SECRET KEY]</value>
+       </property>
+
+       <property>
+         <name>fs.s3.awsSecretAccessKey</name>
+         <value>[AWS SECRET ACCESS KEY]</value>
+       </property>
+
+       <property>
+         <name>fs.s3.awsSessionToken</name>
```

Comment:
@Pscheidl can you confirm this actually works?

Reply:
No, the configuration has to be altered. But it can actually work. Created a fix PR here: https://github.com/h2oai/h2o-3/pull/4352

Thank you.